mssql-plugin/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -42,7 +47,7 @@ InsertBQDataQueryFile=testData/BigQuery/BigQueryInsertDataQuery.txt
 
 datatypeColumns=(ID VARCHAR(100) PRIMARY KEY, COL1 TINYINT, COL2 SMALLINT, COL3 INT, COL4 BIGINT,\
   COL5 BIT, COL6 DECIMAL(38, 0), COL7 DECIMAL(10, 4), COL8 DECIMAL(5, 0), COL9 SMALLMONEY, COL10 MONEY, \
-  COL11 NUMERIC(38, 0), COL12 NUMERIC(10, 4), COL13 FLOAT, COL14 REAL, COL15 DATE, COL20 TIME(0), COL21 CHAR(100), \
+  COL11 NUMERIC(38, 0), COL12 NUMERIC(10, 4),bqOutputMultipleDatatypesSchema, COL14 REAL, COL15 DATE, COL20 TIME(0), COL21 CHAR(100), \
```

Comment:
Revert this change you have altered one of the columns with this change.

Reply:
resolved